src/components/buttonPlus/ButtonPlus.styled.js

```diff
@@ -0,0 +1,21 @@
+import styled from '@emotion/styled';
+
+export const SpanStyled = styled.span`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: ${props =>
+    props.width ? `${props.width}px` :  '28px'};

```

Comment:
if here 28px by default than we don't need to determine in in ButtonPlus values by default